.github/workflows/wait_for_required_workflows.yml

```diff
@@ -11,14 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v32
+        uses: jitterbit/get-changed-files@v1
```

Comment:
This action uses the GitHub API to do the diff, which should use a three dots diff.
We've used it before and removed it since it caused us to hit rate limits.
Since we're now on enterprise account our limit increased from 1000 to 15000 request per hour so I think we can try this again until https://github.com/tj-actions/changed-files/issues/702 is resolved.

Another benefit is that we can do a shallow clone which is faster

Reply:
OK so https://github.com/jitterbit/get-changed-files/issues/19#issuecomment-854657729 works